examples/counter-metrics/counter_metrics.description

```diff
@@ -0,0 +1,5 @@
+// Ballerina supports Observability out of the box, and Metrics is one of the three important aspect of the
+// Observability. To observe ballerina code, the `--observe` flag should be given when starting the service.
```

Comment:
Change 'ballerina' to title case, i.e., 'Ballerina'. Apply elsewhere if applicable.